fe/docker/www-dev.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:lts-alpine
+
+WORKDIR /app
+
+# append node_modules to PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install dependencies
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
Please also copy the gitignored `.env` file

Reply:
Fixed in new commit: 8ffdf9d8def4e27d69b9dfaeaf6d21dcdc414892.